starfish/pipeline/algorithmbase.py

```diff
@@ -28,7 +28,7 @@ def run_with_logging(func):
         """
         def helper(*args, **kwargs):
             result = func(*args, **kwargs)
-            # Scenario 1, Filtering
+            # Scenario 1, Filtering, Registration
```

Comment:
```suggestion
            # Scenario 1, Filtering, ApplyTransform
```

So here's an interesting challenge: how do we retain the provenance of LearnTransform -> ApplyTransform?

If you want to defer that to a second PR, I'm ok with that. If you want to chat about it, let's do that. :)